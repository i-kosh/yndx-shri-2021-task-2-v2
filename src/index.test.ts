import { prepareData } from "./index";
import { Entity } from "./types";
const data = require("./__mockdata__/input.json") as Entity[];

describe("Всегда возвращает ожидаемый формат данных", () => {
  it("если не переданы никакие аргументы", () => {
    expect(prepareData()).toMatchSnapshot();
  });

  it("если первый аргумент пустой массив", () => {
    expect(prepareData([])).toMatchSnapshot();
  });

  it("если первый и второй аргументы пустые", () => {
    expect(
      prepareData([], {
        sprintId: 0,
      })
    ).toMatchSnapshot();
  });
});

describe("Функция работает правильно", () => {
  it("Возвращает правильные данные", () => {
    expect(prepareData(data, { sprintId: 977 })).toMatchSnapshot();
  });
});
