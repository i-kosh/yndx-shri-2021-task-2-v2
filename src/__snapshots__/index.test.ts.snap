// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`–í—Å–µ–≥–¥–∞ –≤–æ–∑–≤—Ä–∞—â–∞–µ—Ç –æ–∂–∏–¥–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç –¥–∞–Ω–Ω—ã—Ö –µ—Å–ª–∏ –Ω–µ –ø–µ—Ä–µ–¥–∞–Ω—ã –Ω–∏–∫–∞–∫–∏–µ –∞—Ä–≥—É–º–µ–Ω—Ç—ã 1`] = `
Array [
  Object {
    "alias": "leaders",
    "data": Object {
      "emoji": "üëë",
      "subtitle": "",
      "title": "–ë–æ–ª—å—à–µ –≤—Å–µ–≥–æ –∫–æ–º–º–∏—Ç–æ–≤",
      "users": Array [],
    },
  },
  Object {
    "alias": "vote",
    "data": Object {
      "emoji": "üîé",
      "subtitle": "",
      "title": "–°–∞–º—ã–π üîé –≤–Ω–∏–º–∞—Ç–µ–ª—å–Ω—ã–π —Ä–∞–∑—Ä–∞–±–æ—Ç—á–∏–∫",
      "users": Array [],
    },
  },
  Object {
    "alias": "chart",
    "data": Object {
      "subtitle": "",
      "title": "–ö–æ–º–º–∏—Ç—ã",
      "users": Array [],
      "values": Array [],
    },
  },
  Object {
    "alias": "diagram",
    "data": Object {
      "categories": Array [],
      "differenceText": "",
      "subtitle": "",
      "title": "–†–∞–∑–º–µ—Ä –∫–æ–º–º–∏—Ç–æ–≤",
      "totalText": "",
    },
  },
  Object {
    "alias": "activity",
    "data": Object {
      "data": Object {
        "fri": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "mon": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "sat": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "sun": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "thu": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "tue": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "wed": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
      },
      "subtitle": "",
      "title": "–ö–æ–º–º–∏—Ç—ã, 1 –Ω–µ–¥–µ–ª—è",
    },
  },
]
`;

exports[`–í—Å–µ–≥–¥–∞ –≤–æ–∑–≤—Ä–∞—â–∞–µ—Ç –æ–∂–∏–¥–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç –¥–∞–Ω–Ω—ã—Ö –µ—Å–ª–∏ –ø–µ—Ä–≤—ã–π –∞—Ä–≥—É–º–µ–Ω—Ç –ø—É—Å—Ç–æ–π –º–∞—Å—Å–∏–≤ 1`] = `
Array [
  Object {
    "alias": "leaders",
    "data": Object {
      "emoji": "üëë",
      "subtitle": "",
      "title": "–ë–æ–ª—å—à–µ –≤—Å–µ–≥–æ –∫–æ–º–º–∏—Ç–æ–≤",
      "users": Array [],
    },
  },
  Object {
    "alias": "vote",
    "data": Object {
      "emoji": "üîé",
      "subtitle": "",
      "title": "–°–∞–º—ã–π üîé –≤–Ω–∏–º–∞—Ç–µ–ª—å–Ω—ã–π —Ä–∞–∑—Ä–∞–±–æ—Ç—á–∏–∫",
      "users": Array [],
    },
  },
  Object {
    "alias": "chart",
    "data": Object {
      "subtitle": "",
      "title": "–ö–æ–º–º–∏—Ç—ã",
      "users": Array [],
      "values": Array [],
    },
  },
  Object {
    "alias": "diagram",
    "data": Object {
      "categories": Array [],
      "differenceText": "",
      "subtitle": "",
      "title": "–†–∞–∑–º–µ—Ä –∫–æ–º–º–∏—Ç–æ–≤",
      "totalText": "",
    },
  },
  Object {
    "alias": "activity",
    "data": Object {
      "data": Object {
        "fri": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "mon": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "sat": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "sun": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "thu": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "tue": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "wed": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
      },
      "subtitle": "",
      "title": "–ö–æ–º–º–∏—Ç—ã, 1 –Ω–µ–¥–µ–ª—è",
    },
  },
]
`;

exports[`–í—Å–µ–≥–¥–∞ –≤–æ–∑–≤—Ä–∞—â–∞–µ—Ç –æ–∂–∏–¥–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç –¥–∞–Ω–Ω—ã—Ö –µ—Å–ª–∏ –ø–µ—Ä–≤—ã–π –∏ –≤—Ç–æ—Ä–æ–π –∞—Ä–≥—É–º–µ–Ω—Ç—ã –ø—É—Å—Ç—ã–µ 1`] = `
Array [
  Object {
    "alias": "leaders",
    "data": Object {
      "emoji": "üëë",
      "subtitle": "",
      "title": "–ë–æ–ª—å—à–µ –≤—Å–µ–≥–æ –∫–æ–º–º–∏—Ç–æ–≤",
      "users": Array [],
    },
  },
  Object {
    "alias": "vote",
    "data": Object {
      "emoji": "üîé",
      "subtitle": "",
      "title": "–°–∞–º—ã–π üîé –≤–Ω–∏–º–∞—Ç–µ–ª—å–Ω—ã–π —Ä–∞–∑—Ä–∞–±–æ—Ç—á–∏–∫",
      "users": Array [],
    },
  },
  Object {
    "alias": "chart",
    "data": Object {
      "subtitle": "",
      "title": "–ö–æ–º–º–∏—Ç—ã",
      "users": Array [],
      "values": Array [],
    },
  },
  Object {
    "alias": "diagram",
    "data": Object {
      "categories": Array [],
      "differenceText": "",
      "subtitle": "",
      "title": "–†–∞–∑–º–µ—Ä –∫–æ–º–º–∏—Ç–æ–≤",
      "totalText": "",
    },
  },
  Object {
    "alias": "activity",
    "data": Object {
      "data": Object {
        "fri": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "mon": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "sat": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "sun": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "thu": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "tue": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
        "wed": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
      },
      "subtitle": "",
      "title": "–ö–æ–º–º–∏—Ç—ã, 1 –Ω–µ–¥–µ–ª—è",
    },
  },
]
`;

exports[`–§—É–Ω–∫—Ü–∏—è —Ä–∞–±–æ—Ç–∞–µ—Ç –ø—Ä–∞–≤–∏–ª—å–Ω–æ –í–æ–∑–≤—Ä–∞—â–∞–µ—Ç –ø—Ä–∞–≤–∏–ª—å–Ω—ã–µ –¥–∞–Ω–Ω—ã–µ 1`] = `
Array [
  Object {
    "alias": "leaders",
    "data": Object {
      "emoji": "üëë",
      "subtitle": "–ü–æ—Å–ª–µ–¥–Ω–∏–π –≤–∞–≥–æ–Ω",
      "title": "–ë–æ–ª—å—à–µ –≤—Å–µ–≥–æ –∫–æ–º–º–∏—Ç–æ–≤",
      "users": Array [
        Object {
          "avatar": "8.jpg",
          "id": 8,
          "name": "–ê–ª–µ–∫—Å–∞–Ω–¥—Ä –ò–≤–∞–Ω–∫–æ–≤",
          "valueText": "28",
        },
        Object {
          "avatar": "6.jpg",
          "id": 6,
          "name": "–ê–Ω–¥—Ä–µ–π –ú–æ–∫—Ä–æ—É—Å–æ–≤",
          "valueText": "21",
        },
        Object {
          "avatar": "9.jpg",
          "id": 9,
          "name": "–°–µ—Ä–≥–µ–π –ë–µ—Ä–µ–∂–Ω–æ–π",
          "valueText": "11",
        },
        Object {
          "avatar": "5.jpg",
          "id": 5,
          "name": "–ê–ª–µ–∫—Å–∞–Ω–¥—Ä –ù–∏–∫–æ–ª–∞–∏—á–µ–≤",
          "valueText": "7",
        },
        Object {
          "avatar": "11.jpg",
          "id": 11,
          "name": "–Æ—Ä–∏–π –§—Ä–æ–ª–æ–≤",
          "valueText": "7",
        },
        Object {
          "avatar": "2.jpg",
          "id": 2,
          "name": "–ê–ª–µ–∫—Å–∞–Ω–¥—Ä –®–ª–µ–π–∫–æ",
          "valueText": "6",
        },
        Object {
          "avatar": "4.jpg",
          "id": 4,
          "name": "–í–∞–¥–∏–º –ü–∞—Ü–µ–≤",
          "valueText": "6",
        },
        Object {
          "avatar": "12.jpg",
          "id": 12,
          "name": "–ê–ª–µ–∫—Å–µ–π –Ø—Ä–æ—à–µ–≤–∏—á",
          "valueText": "5",
        },
        Object {
          "avatar": "3.jpg",
          "id": 3,
          "name": "–î–∞—Ä—å—è –ö–æ–≤–∞–ª–µ–≤–∞",
          "valueText": "4",
        },
        Object {
          "avatar": "10.jpg",
          "id": 10,
          "name": "–Ø–Ω–∞ –ë–µ—Ä–Ω–∏–∫–æ–≤–∞",
          "valueText": "4",
        },
        Object {
          "avatar": "7.jpg",
          "id": 7,
          "name": "–î–º–∏—Ç—Ä–∏–π –ê–Ω–¥—Ä–∏—è–Ω–æ–≤",
          "valueText": "3",
        },
        Object {
          "avatar": "1.jpg",
          "id": 1,
          "name": "–ï–≤–≥–µ–Ω–∏–π –î–µ–º–µ–Ω—Ç—å–µ–≤",
          "valueText": "2",
        },
      ],
    },
  },
  Object {
    "alias": "vote",
    "data": Object {
      "emoji": "üîé",
      "subtitle": "–ü–æ—Å–ª–µ–¥–Ω–∏–π –≤–∞–≥–æ–Ω",
      "title": "–°–∞–º—ã–π üîé –≤–Ω–∏–º–∞—Ç–µ–ª—å–Ω—ã–π —Ä–∞–∑—Ä–∞–±–æ—Ç—á–∏–∫",
      "users": Array [
        Object {
          "avatar": "7.jpg",
          "id": 7,
          "name": "–î–º–∏—Ç—Ä–∏–π –ê–Ω–¥—Ä–∏—è–Ω–æ–≤",
          "valueText": "306 –≥–æ–ª–æ—Å–æ–≤",
        },
        Object {
          "avatar": "8.jpg",
          "id": 8,
          "name": "–ê–ª–µ–∫—Å–∞–Ω–¥—Ä –ò–≤–∞–Ω–∫–æ–≤",
          "valueText": "305 –≥–æ–ª–æ—Å–æ–≤",
        },
        Object {
          "avatar": "5.jpg",
          "id": 5,
          "name": "–ê–ª–µ–∫—Å–∞–Ω–¥—Ä –ù–∏–∫–æ–ª–∞–∏—á–µ–≤",
          "valueText": "284 –≥–æ–ª–æ—Å–∞",
        },
        Object {
          "avatar": "4.jpg",
          "id": 4,
          "name": "–í–∞–¥–∏–º –ü–∞—Ü–µ–≤",
          "valueText": "273 –≥–æ–ª–æ—Å–∞",
        },
        Object {
          "avatar": "1.jpg",
          "id": 1,
          "name": "–ï–≤–≥–µ–Ω–∏–π –î–µ–º–µ–Ω—Ç—å–µ–≤",
          "valueText": "270 –≥–æ–ª–æ—Å–æ–≤",
        },
        Object {
          "avatar": "6.jpg",
          "id": 6,
          "name": "–ê–Ω–¥—Ä–µ–π –ú–æ–∫—Ä–æ—É—Å–æ–≤",
          "valueText": "264 –≥–æ–ª–æ—Å–∞",
        },
        Object {
          "avatar": "2.jpg",
          "id": 2,
          "name": "–ê–ª–µ–∫—Å–∞–Ω–¥—Ä –®–ª–µ–π–∫–æ",
          "valueText": "242 –≥–æ–ª–æ—Å–∞",
        },
        Object {
          "avatar": "11.jpg",
          "id": 11,
          "name": "–Æ—Ä–∏–π –§—Ä–æ–ª–æ–≤",
          "valueText": "224 –≥–æ–ª–æ—Å–∞",
        },
        Object {
          "avatar": "9.jpg",
          "id": 9,
          "name": "–°–µ—Ä–≥–µ–π –ë–µ—Ä–µ–∂–Ω–æ–π",
          "valueText": "219 –≥–æ–ª–æ—Å–æ–≤",
        },
        Object {
          "avatar": "3.jpg",
          "id": 3,
          "name": "–î–∞—Ä—å—è –ö–æ–≤–∞–ª–µ–≤–∞",
          "valueText": "216 –≥–æ–ª–æ—Å–æ–≤",
        },
        Object {
          "avatar": "10.jpg",
          "id": 10,
          "name": "–Ø–Ω–∞ –ë–µ—Ä–Ω–∏–∫–æ–≤–∞",
          "valueText": "212 –≥–æ–ª–æ—Å–æ–≤",
        },
        Object {
          "avatar": "12.jpg",
          "id": 12,
          "name": "–ê–ª–µ–∫—Å–µ–π –Ø—Ä–æ—à–µ–≤–∏—á",
          "valueText": "210 –≥–æ–ª–æ—Å–æ–≤",
        },
      ],
    },
  },
  Object {
    "alias": "chart",
    "data": Object {
      "subtitle": "–ü–æ—Å–ª–µ–¥–Ω–∏–π –≤–∞–≥–æ–Ω",
      "title": "–ö–æ–º–º–∏—Ç—ã",
      "users": Array [
        Object {
          "avatar": "8.jpg",
          "id": 8,
          "name": "–ê–ª–µ–∫—Å–∞–Ω–¥—Ä –ò–≤–∞–Ω–∫–æ–≤",
          "valueText": "28",
        },
        Object {
          "avatar": "6.jpg",
          "id": 6,
          "name": "–ê–Ω–¥—Ä–µ–π –ú–æ–∫—Ä–æ—É—Å–æ–≤",
          "valueText": "21",
        },
        Object {
          "avatar": "9.jpg",
          "id": 9,
          "name": "–°–µ—Ä–≥–µ–π –ë–µ—Ä–µ–∂–Ω–æ–π",
          "valueText": "11",
        },
        Object {
          "avatar": "5.jpg",
          "id": 5,
          "name": "–ê–ª–µ–∫—Å–∞–Ω–¥—Ä –ù–∏–∫–æ–ª–∞–∏—á–µ–≤",
          "valueText": "7",
        },
        Object {
          "avatar": "11.jpg",
          "id": 11,
          "name": "–Æ—Ä–∏–π –§—Ä–æ–ª–æ–≤",
          "valueText": "7",
        },
        Object {
          "avatar": "2.jpg",
          "id": 2,
          "name": "–ê–ª–µ–∫—Å–∞–Ω–¥—Ä –®–ª–µ–π–∫–æ",
          "valueText": "6",
        },
        Object {
          "avatar": "4.jpg",
          "id": 4,
          "name": "–í–∞–¥–∏–º –ü–∞—Ü–µ–≤",
          "valueText": "6",
        },
        Object {
          "avatar": "12.jpg",
          "id": 12,
          "name": "–ê–ª–µ–∫—Å–µ–π –Ø—Ä–æ—à–µ–≤–∏—á",
          "valueText": "5",
        },
        Object {
          "avatar": "3.jpg",
          "id": 3,
          "name": "–î–∞—Ä—å—è –ö–æ–≤–∞–ª–µ–≤–∞",
          "valueText": "4",
        },
        Object {
          "avatar": "10.jpg",
          "id": 10,
          "name": "–Ø–Ω–∞ –ë–µ—Ä–Ω–∏–∫–æ–≤–∞",
          "valueText": "4",
        },
        Object {
          "avatar": "7.jpg",
          "id": 7,
          "name": "–î–º–∏—Ç—Ä–∏–π –ê–Ω–¥—Ä–∏—è–Ω–æ–≤",
          "valueText": "3",
        },
        Object {
          "avatar": "1.jpg",
          "id": 1,
          "name": "–ï–≤–≥–µ–Ω–∏–π –î–µ–º–µ–Ω—Ç—å–µ–≤",
          "valueText": "2",
        },
      ],
      "values": Array [
        Object {
          "hint": "–§–µ–≤—Ä–∞–ª—å—Å–∫–∞—è –ª–∞–∑—É—Ä—å",
          "title": "958",
          "value": 460,
        },
        Object {
          "hint": "–ù–æ–≤—ã–π –≥–æ–¥",
          "title": "959",
          "value": 209,
        },
        Object {
          "hint": "–ù–æ–≤—ã–π –∫–æ–¥",
          "title": "960",
          "value": 288,
        },
        Object {
          "hint": "–ü–æ—Å–ª–µ–≤–∫—É—Å–∏–µ",
          "title": "961",
          "value": 258,
        },
        Object {
          "hint": "–° –¥–Ω–µ–º –º—É–∂–∏–∫–∞!",
          "title": "962",
          "value": 152,
        },
        Object {
          "hint": "–ù–æ–≤—ã–π —Å–ø—Ä–∏–Ω—Ç",
          "title": "963",
          "value": 249,
        },
        Object {
          "hint": "–§–µ–µ—á–∫–∏ –Ω–∞ –µ–¥–∏–Ω–æ—Ä–æ–≥–∞—Ö",
          "title": "964",
          "value": 249,
        },
        Object {
          "hint": "–†–∞–∑–¥–∞—á–∞ –¥–æ–ª–≥–æ–≤",
          "title": "965",
          "value": 446,
        },
        Object {
          "hint": "–¢—Ä–∏ –ø–ª—é—Å –¥–≤–∞",
          "title": "966",
          "value": 477,
        },
        Object {
          "hint": "–ú–∞–π—Å–∫–∏–µ –ø—Ä–∞–∑–¥–Ω–∏–∫–∏",
          "title": "967",
          "value": 707,
        },
        Object {
          "hint": "–°–ª–µ–¥—É—é—â–∏–π",
          "title": "968",
          "value": 475,
        },
        Object {
          "hint": "–¢–æ–ø–æ–ª–∏–Ω—ã–π –ø—É—Ö",
          "title": "969",
          "value": 655,
        },
        Object {
          "hint": "–ñ–∞—Ä–∞",
          "title": "970",
          "value": 354,
        },
        Object {
          "hint": "–†–µ–ø–∫–∞",
          "title": "971",
          "value": 253,
        },
        Object {
          "hint": "–†–∏—Å—É–µ–º —Å–æ–≤—É",
          "title": "972",
          "value": 233,
        },
        Object {
          "hint": "–î–≤–∞ –º–µ—Å—Ç–∞",
          "title": "973",
          "value": 170,
        },
        Object {
          "hint": "–î–µ–≤—è—Ç—ã–π –≤–∞–ª",
          "title": "974",
          "value": 182,
        },
        Object {
          "hint": "–õ–µ—Ç–æ –≤—Å–µ",
          "title": "975",
          "value": 160,
        },
        Object {
          "hint": "–°–Ω–æ–≤–∞ –≤ —à–∫–æ–ª—É",
          "title": "976",
          "value": 210,
        },
        Object {
          "active": true,
          "hint": "–ü–æ—Å–ª–µ–¥–Ω–∏–π –≤–∞–≥–æ–Ω",
          "title": "977",
          "value": 104,
        },
        Object {
          "hint": "–ü—Ä–µ–º—å–µ—Ä–∞ —Å–µ–∑–æ–Ω–∞",
          "title": "978",
          "value": 117,
        },
        Object {
          "hint": "–û—Å–µ–Ω–Ω—è—è –ø–æ—Ä–∞",
          "title": "979",
          "value": 122,
        },
        Object {
          "hint": "–ú–∞—à–∞, —Å –¥–Ω–µ–º —Ä–æ–∂–¥–µ–Ω–∏—è!",
          "title": "980",
          "value": 85,
        },
        Object {
          "hint": "–ù–æ–≤–∞—è –∫–æ–ª–µ—è",
          "title": "981",
          "value": 90,
        },
        Object {
          "hint": "F5",
          "title": "982",
          "value": 123,
        },
        Object {
          "hint": "–ü—É—à–Ω–æ–π –∑–≤–µ—Ä–µ–∫",
          "title": "983",
          "value": 129,
        },
        Object {
          "hint": "–†–æ–∑–æ–≤—ã–π —Å–ª–æ–Ω",
          "title": "984",
          "value": 120,
        },
        Object {
          "hint": "–í–µ—Å–Ω–∞ –∏ –∂–µ–Ω—â–∏–Ω—ã",
          "title": "985",
          "value": 173,
        },
        Object {
          "hint": "–¢—Ä–µ—Ç–∏–π —Å –∫–æ–Ω—Ü–∞",
          "title": "986",
          "value": 122,
        },
        Object {
          "hint": "–°–º–∏—Ä–µ–Ω–∏–µ",
          "title": "987",
          "value": 126,
        },
        Object {
          "hint": "–ù–æ–≤–æ–≥–æ–¥–Ω–∏–π –∑–∞—Ç—è–∂–Ω–æ–π",
          "title": "988",
          "value": 131,
        },
        Object {
          "hint": "–õ–∏—Å—Ç–æ–ø–∞–¥",
          "title": "989",
          "value": 119,
        },
        Object {
          "hint": "–ê—Ä–±—É–∑–∏–∫–∏",
          "title": "990",
          "value": 79,
        },
        Object {
          "hint": "–ö–æ—Ä–æ—Ç–∫–∏–π –ö—Ä–æ—Ç",
          "title": "991",
          "value": 28,
        },
        Object {
          "hint": "–ö–∞—Ä–∞–Ω—Ç–∏–Ω",
          "title": "992",
          "value": 0,
        },
        Object {
          "hint": "–°–¥–∞–µ–º —Ö–≤–æ—Å—Ç—ã",
          "title": "993",
          "value": 0,
        },
        Object {
          "hint": "–ö—Ä–∞—Å–Ω—ã–π",
          "title": "994",
          "value": 0,
        },
        Object {
          "hint": "–°–∞–ª–∞—Ç–∏–∫–∏",
          "title": "995",
          "value": 0,
        },
        Object {
          "hint": "–ë–∞—Ä–º–∞–ª–µ–π",
          "title": "996",
          "value": 0,
        },
      ],
    },
  },
  Object {
    "alias": "diagram",
    "data": Object {
      "categories": Array [
        Object {
          "differenceText": "-3 –∫–æ–º–º–∏—Ç–∞",
          "title": "> 1001 —Å—Ç—Ä–æ–∫–∏",
          "valueText": "2 –∫–æ–º–º–∏—Ç–∞",
        },
        Object {
          "differenceText": "-3 –∫–æ–º–º–∏—Ç–∞",
          "title": "501 ‚Äî 1000 —Å—Ç—Ä–æ–∫",
          "valueText": "3 –∫–æ–º–º–∏—Ç–∞",
        },
        Object {
          "differenceText": "-22 –∫–æ–º–º–∏—Ç–∞",
          "title": "101 ‚Äî 500 —Å—Ç—Ä–æ–∫",
          "valueText": "13 –∫–æ–º–º–∏—Ç–æ–≤",
        },
        Object {
          "differenceText": "-78 –∫–æ–º–º–∏—Ç–æ–≤",
          "title": "1 ‚Äî 100 —Å—Ç—Ä–æ–∫",
          "valueText": "86 –∫–æ–º–º–∏—Ç–æ–≤",
        },
      ],
      "differenceText": "-106 —Å –ø—Ä–æ—à–ª–æ–≥–æ —Å–ø—Ä–∏–Ω—Ç–∞",
      "subtitle": "–ü–æ—Å–ª–µ–¥–Ω–∏–π –≤–∞–≥–æ–Ω",
      "title": "–†–∞–∑–º–µ—Ä –∫–æ–º–º–∏—Ç–æ–≤",
      "totalText": "104 –∫–æ–º–º–∏—Ç–∞",
    },
  },
  Object {
    "alias": "activity",
    "data": Object {
      "data": Object {
        "fri": Array [
          0,
          0,
          4,
          0,
          1,
          0,
          0,
          3,
          1,
          0,
          3,
          1,
          1,
          4,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          2,
        ],
        "mon": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          11,
          0,
          0,
          0,
          0,
          0,
        ],
        "sat": Array [
          0,
          0,
          0,
          0,
          0,
          3,
          0,
          0,
          0,
          6,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          0,
        ],
        "sun": Array [
          0,
          2,
          1,
          1,
          1,
          1,
          1,
          2,
          0,
          1,
          1,
          2,
          4,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
          1,
          0,
          0,
          0,
        ],
        "thu": Array [
          0,
          0,
          6,
          2,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          1,
          0,
          0,
          1,
          0,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
        ],
        "tue": Array [
          0,
          0,
          0,
          3,
          0,
          1,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          1,
          0,
          0,
          0,
          0,
          4,
          1,
          0,
          0,
        ],
        "wed": Array [
          0,
          0,
          4,
          0,
          0,
          0,
          0,
          0,
          0,
          1,
          0,
          1,
          1,
          0,
          2,
          0,
          0,
          0,
          3,
          1,
          1,
          1,
          1,
          1,
        ],
      },
      "subtitle": "–ü–æ—Å–ª–µ–¥–Ω–∏–π –≤–∞–≥–æ–Ω",
      "title": "–ö–æ–º–º–∏—Ç—ã, 1 –Ω–µ–¥–µ–ª—è",
    },
  },
]
`;
